composer.json

```diff
@@ -1,14 +1,18 @@
 {
-	"name": "chrisboulton/php-resque",
+	"name": "spiritdead/php-resque",
```

Comment:
This change cannot be merged.  Always be sure to use a dedicated branch for PRs, and not to commit any changes to a PR branch that you don't intend to submit in the PR itself, as GitHub PRs are per branch, not per change set.